README.md

```diff
@@ -28,8 +28,10 @@ You can delete `/etc/pacman.d/mirrorlist.asahi` now.
 
 Remove the old keyring package:
 
+Install the new keyring package (will replace the legacy `asahilinux-keyring`):
+
 ```
-sudo pacman -Rdd asahilinux-keyring
+sudo pacman -S asahi-alarm-keyring
```

Comment:
is this right? The PKGBUILD does not specify conflicts (yet - we should probably do this though)

Reply:
I see you already did 👍